specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2021-03-01-preview/Settings.json

```diff
@@ -0,0 +1,594 @@
+            "$ref": "#/parameters/SettingsName"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/Settings"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../common/1.0/types.json#/definitions/CloudError"
+            }
+          }
+        }
+      },
+      "delete": {
+        "x-ms-examples": {
+          "Delete EyesOn settings.": {
+            "$ref": "./examples/settings/DeleteEyesOnSetting.json"
+          }
+        },
+        "tags": [
+          "Settings"
+        ],
+        "description": "Delete setting of the product.",
+        "operationId": "ProductSettings_Delete",
+        "parameters": [
+          {
+            "$ref": "../../../common/1.0/types.json#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "../../../common/1.0/types.json#/parameters/SubscriptionId"
+          },
+          {
+            "$ref": "../../../common/1.0/types.json#/parameters/ResourceGroupName"
+          },
+          {
+            "$ref": "../../../common/1.0/types.json#/parameters/OperationalInsightsResourceProvider"
+          },
+          {
+            "$ref": "../../../common/1.0/types.json#/parameters/WorkspaceName"
+          },
+          {
+            "$ref": "#/parameters/SettingsName"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK"
+          },
+          "204": {
+            "description": "No Content"
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../common/1.0/types.json#/definitions/CloudError"
+            }
+          }
+        }
+      },
+      "put": {
+        "x-ms-examples": {
+          "Update EyesOn settings.": {
+            "$ref": "./examples/settings/UpdateEyesOnSetting.json"
+          }
+        },
+        "tags": [
+          "Settings"
+        ],
+        "description": "Updates setting.",
+        "operationId": "ProductSettings_Update",
+        "parameters": [
+          {
+            "$ref": "../../../common/1.0/types.json#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "../../../common/1.0/types.json#/parameters/SubscriptionId"
+          },
+          {
+            "$ref": "../../../common/1.0/types.json#/parameters/ResourceGroupName"
+          },
+          {
+            "$ref": "../../../common/1.0/types.json#/parameters/OperationalInsightsResourceProvider"
+          },
+          {
+            "$ref": "../../../common/1.0/types.json#/parameters/WorkspaceName"
+          },
+          {
+            "$ref": "#/parameters/SettingsName"
+          },
+          {
+            "$ref": "#/parameters/Settings"
```

Comment:
Is this body? Don't you need 
```         
            "in": "body",
            "required": true,

Reply:
Yes, It's inside the schema
![image](https://user-images.githubusercontent.com/28350648/108342019-f9a7bf80-71e2-11eb-9784-8289dfe33cfc.png)